.gitignore

```diff
@@ -85,5 +85,4 @@ config/crd/bases/kataconfiguration.openshift.io_kataconfigs.yaml
 config/rbac/role.yaml
 config/webhook/manifests.yaml
 # make bundle files
-bundle.Dockerfile
 bundle/
```

Comment:
Missing newline since commit 846b467294adb3aa0bd23294389339835d7162a4. Maybe fix this as well while here ?